weekly-updates/weekly-update.2015-10-16.md

```diff
@@ -0,0 +1,54 @@
+layout: blog-post.hbs
+---
+
+### Node.js News — October 16th
+Node.js v4.2.1 (LTS), v4.2.0 (LTS) are released
+
+### Node.js v4.2.1 (LTS) and v4.2.0 (LTS) Releases
+
+This week we have two releases: [Node.js v4.2.1 (LTS)](https://nodejs.org/en/blog/release/v4.2.1/) and [Node.js v4.2.0 (LTS)](https://nodejs.org/en/blog/release/v4.2.0/). complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+
+### Notable changes
+
+* Includes fixes for two regressions
+  - Assertion error in WeakCallback  - see [#3329](https://github.com/nodejs/node/pull/3329)
+  - Undefined timeout regression - see [#3331](https://github.com/nodejs/node/pull/3331)
+
+### Known issues
+
+* When a server queues a large amount of data to send to a client over a pipelined HTTP connection, the underlying socket may be destroyed. See [#3332](https://github.com/nodejs/node/issues/3332) and [#3342](https://github.com/nodejs/node/pull/3342).
+* Some problems with unreferenced timers running during `beforeExit` are still to be resolved. See [#1264](https://github.com/nodejs/node/issues/1264).
+* Surrogate pair in REPL can freeze terminal. [#690](https://github.com/nodejs/node/issues/690)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/node/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/node/issues/1435).
+
+### Community Updates
+
+* [Node.js v4.2.1 Release Summary and Stats](https://nodesource.com/blog/nodejs-v421-release-summary-and-stats), "Closely tracking the Long Term Support plan, 4.2.0 and 4.2.1 have been released this week as LTS. v4.2.1 has been released to fix two critical regressions in v4.2.0. It is highly suggested you use v4.2.1 instead of v4.2.0"
+* [A Hubot Plugin that Helps You to Choose Where to Drink Coffee](http://ditrospecta.com/javascript/plugin/hubot/2015/10/03/hubot-plugin-choose-place-to-drink-coffee.html), "Hubot is a very friendly and easy extensible bot written in Coffeescript"
+* [Building using Node-GYP with Visual Studio Express 2015 on Windows 10 Pro x64](http://www.serverpals.com/blog/building-using-node-gyp-with-visual-studio-express-2015-on-windows-10-pro-x64), "Setting up a development environment, using Node-GYP with Visual Studio Express 2015 on Windows 10 Pro x64"
+* [Linux and Node.js in Production using Daemontools and NDT](http://www.serverpals.com/blog/linux-nodejs-in-production-using-daemontools-ndt), "Putting the application to the test with Linux and Node.js in Production using Daemontools and NDT"
+* [Using Bower with BowerCDN for better web performance!](http://www.serverpals.com/blog/using-bower-with-bowercdn-for-better-web-performance), "BowerCDN is a service built on the bower API that installs and maintains permalinks to bower repository content"
+
```

Comment:
I don't think we need all three serverpals.com links in one update. I would just pick the best one or the most recent one.